pyodide_build/pywasmcross.py

```diff
@@ -212,6 +226,20 @@ def handle_command(line, args, dryrun=False):
             return
         if arg.startswith('/tmp'):
             return
+        # automake generated configure scripts:
+        if 'DUALCASE' in env and 'as_nl' in env:
+            # It would be really nice if there was a clear sign
+            # in the environment, but there isn't
+            if arg in ["-qversion", "--version", "--help", "-v", "-V"]:
+                return
+            if arg.startswith("-print-"):
+                return
+            if arg in ["conftest.cpp", "conftest.c", "@conftest.lst"]:
```

Comment:
Are those specific to automake (not too familiar with it)? Maybe some comment could be useful.